AcloConcurrency/AcloConcurrency/AlcoConcurrencyApp.swift

```diff
@@ -2,22 +2,18 @@ import SwiftUI
 
 @main
 struct AlcoConcurrencyApp: App {
-    
-    @StateObject private var collectionsViewModel: CollectionsViewModel
-    
+
+    @StateObject private var place: PlacesViewModel
```

Comment:
Add space between init and variables
And please use fullName placesViewModel. You can use(if you want) name just "viewModel" in your view or something, but not in Main Struct.

Reply:
how can I get fck linter ....